src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,16 @@
+import { useContext } from 'react';
+import { TodoItem } from '../TodoItem';
+import './style.css';
+import { VisibleTodosContext } from '../GlobalStateProvider';
+
+export const TodoList: React.FC = () => {
+  const todos = useContext(VisibleTodosContext);
+
+  return (
+    <ul className="todo-list" data-cy="todoList">
```

Comment:
```suggestion
    <ul className="todo-list" data-cy="todosList">
```
I know you kept the original data-cy attribute as required. It's not your fault. The original data-cy attribute is wrong - known issue - it'll be fixed.

But for now you have to make this change to pass tests